### ISSUE TITLE
[React Native Gallery -> VirtualizedList]: Narrator fails to announce the name of the combo box.

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-gallery#371]
**Pre-Requisite:**
Turn on Narrator (Ctrl + Win + Enter).

**Repro-Steps:**

Launch the React Native Gallery app.
Navigate to VirtualizedList’ button and invoke it.
Navigate to 'Selection Support' combo box present under "A VirtualizedList with multiple selection support" group.
Listen to narrator announcement.

**Actual Result:**

Narrator fails to announce the name of the combo box. When user navigate to 'Selection Support' combo box, narrator announces as "None, combo box, collapsed".

**Expected Result:**
Screen reader should the name of the combo box. When user navigate to 'Selection Support' combo box, narrator should announce as "Selection support, combo box, collapsed".

**User Impact:**
Users with visual impairments who rely on screen reader will not be able to know about the combo box name if narrator fails to announce about it.

Internal Bug Link: https://dev.azure.com/microsoft/OS/_workitems/edit/47851722
